ecs-task-definition-deploy/action.yml

```diff
@@ -0,0 +1,123 @@
+name: Determine Python and Poetry versions
+description: Determine Python and Poetry versions
+
+inputs:
+  application_id:
+    description: Application ID also the task definition name in this case
+    required: true
+  image_uri:
+    description: The URI of the container image to insert into the ECS task definition
+    required: true
+  deregister_previous:
+    description: Determines whether to deregister previous active task definition
+    required: false
+    default: "false"
```

Comment:
Since it is a one line `aws` command with no extra logics to be done, there is no need creating a separate action
We can deregister just as after we have successfully updated the service